examples/concurrency/test/concurrency_test.dart

```diff
@@ -4,9 +4,11 @@ import 'package:concurrency_examples/async_number_of_keys.dart'
     as async_number_of_keys;
 import 'package:concurrency_examples/simple_worker_isolate.dart'
     as simple_worker_isolate;
+import 'package:concurrency_examples/simple_isolate_closure.dart'
+    as simple_isolate_closure;
 import 'package:test/test.dart';
 
-void main() {
+Future<void> main() async {
```

Comment:
I removed `async` here and got this error from `dart analyze`:

```
  error • test/concurrency_test.dart:11:14 • The body might complete normally, causing 'null' to be returned, but the return
          type, 'Future<void>', is a potentially non-nullable type. Try adding either a return or a throw statement at the
          end. • body_might_complete_normally
```

Nbd since it's just a nit, but any thoughts on that?

Reply:
If the function is no longer `async`, you can just return `void` instead of `Future<void>`.